proto/public/uber/cadence/api/v1/error.proto

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2020 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+syntax = "proto3";
+
+package uber.cadence.api.v1;
+
+option go_package = "apiv1";
+option java_multiple_files = true;
+option java_outer_classname = "ApiProto";
+option java_package = "com.uber.cadence.api.v1";
+
+message WorkflowExecutionAlreadyStartedError {
+  string start_request_id = 1;
+  string run_id = 2;
+}
+
+message EntityNotExistsError {
+  string current_cluster = 1;
+  string active_cluster = 2;
+}
+
+message DomainNotActiveError {
+  string domain = 1;
+  string current_cluster = 2;
+  string active_cluster = 3;
+}
+
+message RetryTaskV2Error {
+  string domain_id = 1;
+  string workflow_id = 2;
+  string run_id = 3;
```

Comment:
similar to above: there's both `workflow_id + run_id` field pairs and `api.v1.WorkflowExecution` used in what seem like inconsistent ways, with a handful of exceptions where only `workflow_id` is relevant or where the whole request is just these two fields (so nesting seems unnecessary).  standardize?  or is there a reason they're separate in some places?

Reply:
I like this proposal, it would be more consistent. Unless team have any semantic reasons why were declared separately in the first place.

Reply:
Updated to wrap all `workflow_id + run_id` pairs with `api.v1.WorkflowExecution`.